milli/src/update/settings.rs

```diff
@@ -411,6 +422,21 @@ impl<'a, 't, 'u, 'i> Settings<'a, 't, 'u, 'i> {
         }
     }
 
+    fn update_exact_attributes(&mut self) -> Result<bool> {
+        match self.exact_attributes {
+            Setting::Set(ref attrs) => {
```

Comment:
Is it required, even in recent versions of Rust?
```suggestion
            Setting::Set(attrs) => {
```

Reply:
yes still necessary